Map Page UI Updates

mwodahl opened the pull request:
# PR Details

This PR updates the map page to include the UI updates found in the following Figma prototype: https://www.figma.com/proto/QG64ugNXpXMUk2vQ1wgPdZ/CV-Manager-Prototype-January-2025?page-id=2143%3A6061&node-id=2146-7749&viewport=-595%2C-659%2C0.22&t=A1CuORABoO0Qexzk-1&scaling=scale-down&content-scaling=fixed&starting-point-node-id=2146%3A6646

## Description

This PR includes the following changes:
- Update menu opacity and background color
- Update RSU popup to include color-coded information and icon
- Update RSU configuration menus (single and multiple) with correct layout, button types
- Update the counts menu input display
- Update the RSU Status component to include color-coded information, location icon, and correct button variations

## How Has This Been Tested?

This was tested locally using docker compose alongside the changes found in #172. Unit tests were verified to pass using npm.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ X ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] My changes require new environment variables:
  - [ ] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [ ] My changes require updates to the documentation:
  - [ ] I have updated the documentation accordingly.
- [ X ] My changes require updates and/or additions to the unit tests:
  - [ X ] I have modified/added tests to cover my changes.
- [ X ] All existing tests pass.

jacob6838 committed:
Adding client side report generation

jacob6838 committed:
Creating CountWithId type

jacob6838 committed:
Re-name formatNumber to formatAxisTickNumber

jacob6838 committed:
Fixing report refresh

jacob6838 committed:
reportColorPallettes converted to named object

jacob6838 committed:
Removing single-use reports/graph components

jacob6838 committed:
Re-working laneDirectionOfTravelReportData to map

jacob6838 committed:
Update webapp/src/features/intersections/reports/report-list-table.tsx

Co-authored-by: Matt Cook <mattheworion.cook@gmail.com>

jacob6838 committed:
Update ReportsPage.tsx

jacob6838 committed:
Cleaning up unused variables/comments

jacob6838 committed:
Merge branch 'develop' into client-side-intersection-reports

jacob6838 committed:
Updating intersection-api local run command

jacob6838 committed:
Merge branch 'develop' into client-side-intersection-reports

mwodahl committed:
Update menu css style properties

mwodahl committed:
Remove unused css classes

mwodahl committed:
Remove map unused css classes

mwodahl committed:
Remove background color from toggle button

mwodahl committed:
Update menu background color, add configure rsu accordion

mwodahl committed:
Update table format to match UI prototype

mwodahl committed:
Update search field alignment

mwodahl committed:
Remove GenerateRSUErrorsPDF component

mwodahl committed:
Update menu min width

mwodahl committed:
Update counts display to match UI prototype

mwodahl committed:
Update RSU popup to match UI prototype

mwodahl committed:
Move button position, remove title

mwodahl committed:
Remove header, update button

mwodahl committed:
Update input fields to use MUI components, update buttons

mwodahl committed:
Update snmp walk item background color

mwodahl committed:
Move button position, update button style

mwodahl committed:
Update header style, content

mwodahl committed:
Update testing snapshots

mwodahl committed:
Update button icons

mwodahl committed:
Add header margin

mwodahl committed:
Fix configure button RSU width

mwodahl committed:
Snapshot update

jacob6838 committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

jacob6838 committed:
Merge branch 'develop' into client-side-intersection-reports

jacob6838 commented:
A few visual comments to start. Please excuse the use of paint, it was easier this way:
<img width="1917" alt="image" src="https://github.com/user-attachments/assets/a70b71dd-31a6-46eb-9bd3-1371a4ed2320" />

jacob6838 commented:
There is a lot of extra spacing in the RSU side menu. Reducing margins/padding would open up a lot more space
![image](https://github.com/user-attachments/assets/1f8b629b-ddc1-46d7-bbcc-4056046f1ee0)

jacob6838 commented:
RSU side menu scrollbar should freeze the header and scroll only the content: 
![image](https://github.com/user-attachments/assets/6d40cdb5-6b9a-45ae-a3fb-f292f6553db6)

jacob6838 commented:
The selected RSU side menu and "Display message Counts"/"Display RSU Status" display over each other

jacob6838 commented:
Let's add a backlog task to convert the "Configure RSUs" menu to use the same polygon rendering as the V2X message viewer and upcoming mooveai region selector

jacob6838 reviewed (changes requested):
A few high level comments upon testing - will review the code shortly!

jacob6838 committed:
Cleaning up report table and logging

mwodahl committed:
Remove unused css class

mwodahl committed:
Update ConfigureRSU height, scrolling

mwodahl committed:
Update map menu height

mwodahl committed:
Update map menu to be expanded by default

mwodahl committed:
Update menu container height

mwodahl committed:
Map testing snapshot update

mwodahl committed:
Merge branch 'develop' into Update/webapp-map-ui

mwodahl commented:
I've updated the following:
- Map menu is expanded by default
- Removed extra spacing from RSU config menu
- RSU config menu only has scroll bar for content
- Added a backlog item to swap the RSU Config component to use the same polygon selector as the V2x message viewer

jacob6838 committed:
Documenting generatePdf method

mwodahl committed:
Remove typography font override

rgelmis commented:
Padding needs to be consistent on both sides (see yellow lines here) 
![image](https://github.com/user-attachments/assets/187070de-347a-429e-a549-02ca27bd0d8a)

rgelmis commented:
Triangle pointer on map popup needs to match the color of the popup body. I believe the class is `mapboxgl-popup-tip` and `border-bottom-color` is the property to be updated. 
![image](https://github.com/user-attachments/assets/add3b415-4f44-4491-861e-a1026a72484f)

rgelmis commented:
`border-radius` needs to be updated for all of the container elements on the map. I recommend updating them all to `4px`
![image](https://github.com/user-attachments/assets/4364fad7-0cc8-4cf9-8825-facdc69b9dc6)
![image](https://github.com/user-attachments/assets/8af051f6-4cf4-4c3c-9eaa-181693bbcfb3)

rgelmis commented:
Under message forwarding- these inputs should be side-by-side and both should have labels. The label is missing for Destination IP as well 
![image](https://github.com/user-attachments/assets/0fd85158-5b4d-431d-baa9-a67541817aaa)

rgelmis commented:
Instances where we are using empty grid elements for spacing need to be removed and updated to use margins and/or padding instead. This keeps the UI flexible and keeps us from having to update individual grid elements if anything needs to change in the future.
![image](https://github.com/user-attachments/assets/55ed22c2-bb53-4e5a-800f-acde61bb35bd)
![image](https://github.com/user-attachments/assets/6f2197eb-1cf0-4699-abd7-e3f7ff93d77e)

rgelmis commented:
![image](https://github.com/user-attachments/assets/69273cf2-7443-4f94-9818-1111b7c47be2)
Padding should be the same on the right and left sides here, and should be the same for the header and the accordions. This element should also be resized to a width of 400px like in the mockup: 
![image](https://github.com/user-attachments/assets/e895e545-6efa-4223-9f80-b381e9080b0d)

rgelmis commented:
Map Controls > Display V2x Message Viewer: 
![image](https://github.com/user-attachments/assets/9cff022b-0dd4-4e62-81c5-0cb2724c3886)
1. This element has some custom margins that are causing it to not match the sizing of the date selection inputs
2. The submit button has a custom margin that adds too much space to the bottom of the container, this styling should be removed so the top and bottom margins match
3. The submit button should be left-aligned to match other buttons in the UI like these: 
![image](https://github.com/user-attachments/assets/c2869c09-b5ac-4c1b-b2fb-85122ec60186)
4. I recommend adding a label to the dropdown here so it's clear what is being selected 
![image](https://github.com/user-attachments/assets/cf72e59e-a5be-4002-aba2-e1d6e08d6cdc)
5. The spacing between the sidebar and the v2X Message Viewer element is changed on submit and will need to be updated:
![image](https://github.com/user-attachments/assets/1af1ce68-1485-4d8d-a660-4e24420a009c)

rgelmis commented:
## Sidebar Updates
#### Reorganize sidebar into these categories: Map Layers, Filter RSUs, RSU Configuration. Display RSU Status and Display Message Counts should be removed from the sidebar and added to the top right corner as buttons.
### Map Layers 
Includes these as checkboxes: RSU Viewer, Heatmap, V2X Msg Viewer (replaces V2X msg viewer under map controls), WZDx Viewer, Intersections
### Filter RSUs
Filter by vendor (dropdown)
RSU status- note that making any selection under here will automatically "check" RSU Viewer under Map Layers if it is unchecked. 
###### Some additional functionality: Remove the "none" status entirely and default to showing "online" status, allow for selecting between online and SCMS. 
### RSU Configuration
Add points, Configure RSUs (button)
![image](https://github.com/user-attachments/assets/823d2934-9746-4227-94ef-502637206c6c)

### Display RSU Status and Display Message Counts
Move  to the top right corner and display as buttons, reword "Display Message Counts" to "Display Counts" . Only one pane should be visible at a time
![image](https://github.com/user-attachments/assets/f98691a4-d300-490e-9a9f-afb0d88b6043)


## Style Updates
### Accordion Behavior
Clicking an accordion makes the border above it disappear. Ex- Clicking "Map Controls" makes the border under "Layers" disappear. This needs to be fixed so there is clear delineation between accordions
![image](https://github.com/user-attachments/assets/582df1ab-928e-4a5f-af1e-15f64d024e91)

### Select Vendor Dropdown
Remove extra padding, should not be a list item `<li>` since it is not a child of a list element like `<ul>` or `<ol>`
![image](https://github.com/user-attachments/assets/b66fbf6c-bb42-4ed9-affa-7e3edcda65a2)

### Change headers for accordions to font size 20px or 1.25rem
Currently, RSU Status has the correct styling so you could pull from that 
![image](https://github.com/user-attachments/assets/012761f8-b853-4f37-8c99-f0d7bf3741b6)

### Configure RSUs button styling
Update button styling, remove "x" next to "Add Points" and update typography 
![image](https://github.com/user-attachments/assets/3fa6afa4-2a78-41b8-bec1-94f067414f4c)

rgelmis commented:
## RSU Status Updates
### Update Styling
#### Typography
Header: Trebuchet font size 16px (I believe this is text-medium?)
Column headers: Trebuchet size 16px
Badges: Trebuchet size 13px (font-size: small)
SCMS Status: Trebuchet size 13px for status (Unhealthy vs Healthy), size 12px for description
Pagination needs to be updated to Trebuchet as well

#### Buttons
Remove custom hover state, update button labels to match prototype, remove extraneous tooltip
![image](https://github.com/user-attachments/assets/f6c4c0d2-f6e1-4d4a-87d8-6e2a98660c41)
![image](https://github.com/user-attachments/assets/a2471cf5-e566-45fc-adf6-0e71cbafd4b4)

#### Badges

#### Color
"Unhealthy" text should be `#FF6D57`, badge color can stay the same though

#### Layout
Remove min-height and extra padding from top bar 
![image](https://github.com/user-attachments/assets/c145bafe-4ed6-4eb4-8f15-bc0817c0306c)


Update styling for locations to indicate clickability as seen in prototype. On hover, these should underline, similar to links
![image](https://github.com/user-attachments/assets/ad91e54f-6a81-4f50-a4cf-43a935fab19a)


### Update width
Per prototype- width should be 570px

rgelmis commented:
## Display Counts Updates

### Typography
Update all text to Trebuchet
Header: 16px (`text-size: medium`)

### Inputs
Add label to Message Type input, update to `text-align: left` 
![image](https://github.com/user-attachments/assets/a990df91-2429-4f26-86fa-8e444e624d73)

### Table
Column headers should be left-aligned, not center

### Layout
Update width to 400px per prototype

### Other
This might be too much of a logic change to apply in this PR, but the header doesn't need to change based on message type selected. It can remain "Message Counts". Whatever is in the input for Message Type serves as enough of an indicator

jacob6838 committed:
Adding report-utils unit tests

jacob6838 committed:
Adding pdf-generator unit tests

jacob6838 committed:
Adding client side report webapp unit tests

jacob6838 committed:
Updating date-fns library version

jacob6838 committed:
Finalizing report-list-table unit tests

jacob6838 committed:
Mocking date-fns parse in unit tests

jacob6838 committed:
Updating report-list-table tests

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

jacob6838 committed:
Merge branch 'develop' into client-side-intersection-reports

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

mwodahl committed:
Add overrides for action, toolbar on material table

mwodahl committed:
Update destination IP / message type input to display in a grid

mwodahl committed:
Set ConfigureRSU width, remove grid spacing elements

mwodahl committed:
Add label to message type input

mwodahl committed:
Update table actions, location hover display, and error color

mwodahl committed:
Add map-control-container class

mwodahl committed:
add map-control classes, set mapboxgl triangle pointer color

mwodahl committed:
Add border-radius

mwodahl committed:
Update menu component css values

mwodahl committed:
Remove unnecessary V2X message viewer logic

mwodahl committed:
Add styled button for floating buttons on map

mwodahl committed:
Add class to update button font

mwodahl committed:
Remove map controls from menu, add floating buttons for rsu status/counts display, add missing labels to inputs

mwodahl committed:
Remove comment

mwodahl committed:
Add museo-slab class to delete forwarding button

mwodahl committed:
Update outdated testing snapshots

mwodahl commented:
@rgelmis I believe I've addressed each of your comments, but please let me know if I missed any. The changes I've made include:
- Making padding consistent
- Updating mapboxgl popup triangle pointer color
- Added border-radius to all map controls and menus
- Message Forwarding inputs display next to each other
- Removed padding from V2x message viewer, updated button alignment
- Sidebar now matches prototype
- Added floating buttons for RSU Status, Counts components
- RSU Status component has correct error color, toolbar/actions now match prototype
- RSU Counts width/typography is updated to match prototype

payneBrandon committed:
Swapping to profiles for run-intersection option

mwodahl committed:
Merge pull request #188 from CDOT-CV/chore/tasks-update

Swapping to profiles for run-intersection option

payneBrandon commented on webapp/src/pages/Map.tsx:
```diff
@@ -923,177 +898,122 @@ function MapPage() {
+            </FormControl>
+            <Typography sx={{ marginTop: '12px' }}>RSU Status</Typography>
+            <FormControl sx={{ ml: 2, mt: 1 }}>
+              <RadioGroup value={displayType} onChange={handleRsuDisplayTypeChange}>
```
question: we start with neither online nor scms status selected, but this is a radiogroup. Should we allow a "no-select" option here as well?

mwodahl replied:
This was intended to have 'online' selected by default and is now updated

payneBrandon reviewed (changes requested):
Just the one question, looking good!

rgelmis commented:
## Map Menu Container
- `#menu-container` has a hard-coded `top` value of 175px while the Display RSU Status and Message Count buttons calculate `top` like this: 
![image](https://github.com/user-attachments/assets/58539972-a36d-45da-a6a5-1c378553e406) _but_ that value is hard-coded here 
![image](https://github.com/user-attachments/assets/d313d987-daef-45ac-a67b-fbcfb8c5800f)
and that value is totally arbitrary especially now that we have updated the header. Soooo I am good with hard-coding the `top` value for the map menu and the buttons. I think `top` should be whatever the header height is +25px and the right or left (depending on element) should be 25px. Sorry, super long-winded
- Map menu container background color should be the same color as RSU Status menu
## Sidebar Block
Refers to the container where `id="sideBarBlock"`
- Looks like a lot of the elements that are added into this container conditionally are not being spaced properly, leading to dependency on `position` to keep them from overlapping- some examples of this are the header text, the close button, and the tables. This could be fixed by wrapping the elements in their own `<Box>` elements
- The "x" button is being added to the header of each of these, and is off-center and not spaced correctly. Those values are coming from the `PositionedToggleButton.tsx` component. I recommend removing those values entirely, and using a `<Stack>` to align the header and close button 
## Message Counts
- Can you add a message here under the table header to handle empty states? Something like "No data found for the selected date/time" 
![image](https://github.com/user-attachments/assets/c084eb93-5dfb-4f0d-a68e-bdf826bccbbe)
## RSU Config Container
- Add label for Destination IP input
- Recommend updating this container to use grid instead of hard-coded margins to handle spacing
- Destination IP input should have more space, Message Type input can have less
- I believe these accordions were built incorrectly, there shouldn't be `<Accordion>` elements nested in `<Accordion>` elements like they are here: 
![image](https://github.com/user-attachments/assets/75ac89e9-8544-4501-b952-20c0f04aeebc)
Which is causing some weird rendering issues. Recommend removing the `<Accordion>` wrapping the `<AccordionDetails>`
- That should take care of the weird box-shadow/border (?) under the message forwarding description
- Update message forwarding description to use Trebuchet
## RSU Details Block
-Remove hard-coded margins, use grid to handle this
![image](https://github.com/user-attachments/assets/b13c87c9-59e8-4789-8c45-2825992e119d)
- Accordions are built incorrectly here as well, these need to be updated
- I will have you update the layout for the Current Configuration entries in another work item, this PR can be moved forward without making those changes

rgelmis commented:
Oop, one thing I forgot- RSU Status should default to "Online Status" so the RSUs never appear as gray 
![image](https://github.com/user-attachments/assets/32781b65-5204-42d9-a561-c046b7bef988)

mwodahl committed:
Default to RSU online status, remove unused state variables

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

mwodahl committed:
Update layout to use stack, update Destination IP input to have fixed label

mwodahl committed:
Remove unused classes

mwodahl committed:
Update top, left/right position and remove unused class

mwodahl committed:
Update top value

mwodahl committed:
Update headerTabHeight

mwodahl committed:
Add message when no data found to counts table

mwodahl committed:
Update top, right values

mwodahl committed:
Update floating button position value

mwodahl committed:
Add SideBarHeader component for sideBarBlock header

mwodahl committed:
Remove nested accordion components, update accordion elevation

mwodahl committed:
Remove unnecessary class

mwodahl committed:
Remove unnecessary close icon from menu components

mwodahl committed:
Add SideBarHeader, update layout to use stack

mwodahl committed:
Add margin-top to msgfwddiv

mwodahl committed:
Add SideBarHeader to DisplayRsuErrors

mwodahl committed:
Add SideBarHeader to multiple config, remove duplicate accordion elements, add divider between accordion panels

mwodahl committed:
Update outdated testing snapshots

mwodahl committed:
Remove width from multiple RSU config Box

mwodahl committed:
Fix SNMPSet menu button layout on multiple RSU config

mwodahl committed:
Merge branch 'Update/webapp-map-ui' of https://github.com/CDOT-CV/jpo-cvmanager into Update/webapp-map-ui

mwodahl commented:
@rgelmis, I've addressed the second round of comments you left - this is ready for another look!

mwodahl committed:
Remove background color from menu-container

rgelmis reviewed (approved):
this is a work of art, well done picasso 🎨

payneBrandon commented on webapp/src/features/menu/ConfigureRSU.tsx:
```diff
@@ -17,27 +17,14 @@ import '../../components/css/SnmpwalkMenu.css'
+import { SideBarHeader } from '../../styles/components/SideBarHeader'
 
 const ConfigMenu = ({ children }) => {
   const theme = useTheme()
```
question: is this used?

mwodahl replied:
Nope, this has been removed!

Michael7371 committed:
updates to the intersection API dockerfile to use alpine for building the asn1c

Michael7371 committed:
adding missing CV manager API variables to the k8 example scripts

Michael7371 committed:
updating comments in the intersection API dockerfile

Michael7371 committed:
re-adding in maven install to the build step

Michael7371 committed:
adding LOGS_LINK and ENVIRONMENT_NAME environmental variables to the root sample.env

jacob6838 committed:
Not displaying report graphs whe no data, changing report yellow color

Michael7371 committed:
Update services/api/sample.env

Co-authored-by: Matt Cook <mattheworion.cook@gmail.com>

Michael7371 committed:
reverting changes from the api/sample.env and moving them to the root sample.env

Michael7371 committed:
removal of confluent repo file as it is no longer used

payneBrandon reviewed (changes requested):
Talked offline, but the error report print should include RSUs in the error state. Otherwise this looks great!

Michael7371 committed:
Update services/intersection-api/Dockerfile

Co-authored-by: Ivan Yourshaw <39739503+iyourshaw@users.noreply.github.com>

Michael7371 committed:
Update services/intersection-api/Dockerfile

Co-authored-by: Ivan Yourshaw <39739503+iyourshaw@users.noreply.github.com>

mwodahl committed:
Fix RSU error report generation

mwodahl committed:
Remove unnecessary useTheme reference

mwodahl commented:
> Talked offline, but the error report print should include RSUs in the error state. Otherwise this looks great!

This has been updated and now includes RSUs that are in an error state

Michael7371 committed:
Merge pull request #193 from CDOT-CV/Fix/intsection-api-arm64-build-failures

Intersection API Dockerfile Updates to Support ARM64 CPUs

jacob6838 committed:
Labeling report gen form and adding error messages

jacob6838 committed:
Converting report modal to dialog

mwodahl committed:
Merge branch 'Update/ui-theme' into Update/webapp-map-ui

jacob6838 commented:
The green success text on the RSU reboot menu is very difficult to read in dark mode:

<img width="272" alt="image" src="https://github.com/user-attachments/assets/86ef50fc-1611-42a9-87d4-9972980ce89b" />

jacob6838 commented:
The WZDx dialog overflows a bit: 
<img width="328" alt="image" src="https://github.com/user-attachments/assets/7b729776-406e-4ba5-a7f8-91570ff68b97" />

jacob6838 commented:
I noticed some transparent gaps in the map menu as well: 
<img width="205" alt="image" src="https://github.com/user-attachments/assets/f7811edd-ecc0-4953-b242-a2c8950242c6" />

jacob6838 committed:
updating report dialogs to clarify generation timing

jacob6838 committed:
Merge pull request #146 from CDOT-CV/client-side-intersection-reports

Adding client side intersection report generation

mwodahl committed:
Update RSURebootMenu success color

mwodahl committed:
Fix MUI divider color opacity

mwodahl committed:
Remove set width for mapbox popup, update padding values

mwodahl committed:
Add Stack element, set width/height for RSU popup

mwodahl commented:
@jacob6838 I've updated the RSU reboot menu success color, fixed the WZDx popup overflow, and fixed the transparent divider issue

jacob6838 commented:
Ok, last 2 comments:
1. The RSU update menu error text is hard to read:
<img width="307" alt="image" src="https://github.com/user-attachments/assets/b073f2d5-9fa1-43cc-a1db-22dcb28b3034" />
2. The RSU configuration menu hover outline isn't quite in the right spot, and the items are a little close together (top left index is really close to the other entry, and delete button is very low)

jacob6838 committed:
Merge branch 'develop' into Update/webapp-map-ui

mwodahl committed:
Remove height from snmpitemdiv

mwodahl committed:
Update RSUFirmwareMenu to use light text variations for success/error messages

mwodahl committed:
Update outdated snmpwalkitem test snapshot

mwodahl commented:
> Ok, last 2 comments:
> 
> 1. The RSU update menu error text is hard to read:
> 
> <img alt="image" width="307" src="https://private-user-images.githubusercontent.com/51723791/437491190-b073f2d5-9fa1-43cc-a1db-22dcb28b3034.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU1OTMwMjUsIm5iZiI6MTc0NTU5MjcyNSwicGF0aCI6Ii81MTcyMzc5MS80Mzc0OTExOTAtYjA3M2YyZDUtOWZhMS00M2NjLWExZGItMjJkY2IyOGIzMDM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDI1VDE0NTIwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhMTI0ZWY4ODZhNzdhOGI3NjNiM2JjZDgxMTFjNGEzYzUzZWM0M2M3NTkxZjRmZWY5YzNmMzU4ZWJkZWU0YTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Eck-mzIiGSQuWjJOlGKjrADsB8ItebI3lEJRSXkKoRI"> 2. The RSU configuration menu hover outline isn't quite in the right spot, and the items are a little close together (top left index is really close to the other entry, and delete button is very low)

I've updated the RSU update menu error/success text to use the light variants, which makes the text much easier to read. I've also removed the set height from the configuration menu items, which fixes the hover issue and adds more space between items.

jacob6838 reviewed (approved):
Great work, this looks really good!
